### PR TITLE
pallet/asset: New asset extrinsic methods for VC

### PR DESCRIPTION
**Pull request by vatsa287**
This PR,
- Creates 4 new extrinsic methods `vc_create`, `vc_issue`, `vc_transfer` & `vc_status_change`  for VC create & issue.
- Pallet storage changes.
- Asset type changes.
- Unit tests to demonstrate the same.

Signed-off-by: Shreevatsa N <vatsa@dhiway.com>

### CODE REVIEW

**Review by amarts — COMMENTED** [review]
@vatsa287 considering `<VCIssuance>` is a separate one, we need a 'vc_transfer' as a call too. (where you check if the VCInstance is valid, and pick transfer from there). Other things seems fine so far.

**Comment by amarts** [comment]
another thing to work on is, with this code built locally, run `cord --dev` and get asset-tx script to run with newer calls from cord.js

**Comment by vatsa287** [comment]
> @vatsa287 considering `<VCIssuance>` is a separate one, we need a 'vc_transfer' as a call too. (where you check if the VCInstance is valid, and pick transfer from there). Other things seems fine so far.

Added `vc_transfer` & `vc_status_change`. Since type of Instance we store with VCInstance with storage will change.

File: pallets/asset/src/types.rs [hunk 1]
```diff
@@ -38,6 +42,14 @@ pub struct AssetInputEntry<AssetDescription, AssetTypeOf, AssetTag, AssetMeta> {
 	pub asset_meta: AssetMeta,
 }
 
+#[derive(
```
Comment (vatsa287) [inline pallets/asset/src/types.rs]: @amarts I will discard `VCAssetInputEntry` & `VCAssetInputEntryOf` in lib.rs as asset_qty will be taken as an input parameter directly.

**Review by amarts — COMMENTED** [review]
LGTM

**Review by amarts — APPROVED** [review]
LGTM

**Pull request merged**